metals/src/main/scala/scala/meta/internal/metals/watcher/ProjectFileWatcher.scala

```diff
@@ -172,17 +180,10 @@ object ProjectFileWatcher {
           watcher.register(root, Int.MaxValue)
         }
 
-        watcher
+        watcher.close
```

Comment:
It might be cleaner and more idiomatic to define a `FileWatcher` trait and `implicit class` that transforms swoval file watchers into that trait. Then, we could have `SimpleFileWatcher` extend that trait and close them via the same method.

Reply:
Interestingly enough, there is already a FileWatcher interface in this package. It's a little mismatched, but I'll just use that so I don't have to come up with yet another name for a FileWatcher. :sweat_smile:

Reply:
Oh, neat! Yeah, let's do that.